src/meta-srv/src/metasrv.rs

```diff
@@ -225,8 +227,8 @@ impl MetaSrv {
                             );
                             match msg {
                                 LeaderChangeMessage::Elected(_) => {
```

Comment:
More and more code here (Elected and StepDown)，can two methods be added to handle these two events?
`on_leader_start(..)`
`on_leader_stepdown(..)`